cmd/build.go

```diff
@@ -114,16 +115,21 @@ func newBuildCmd() *cobra.Command {
 				logrus.Fatalf("failed to parse meta %s, meta must be formated with JSON: %v", string(metaJSON), err)
 			}
 
-			homedir, err := homedir.Dir()
+			cwd, err := os.Getwd()
 			if err != nil {
 				logrus.Fatal(err)
 			}
 
-			sdlocalDir := filepath.Join(homedir, ".sdlocal")
-			cwd, err := os.Getwd()
-			if err != nil {
-				logrus.Fatal(err)
+			configBaseDir := cwd
+
+			if !useLocalConfig {
+				configBaseDir, err = homedir.Dir()
```

Comment:
I think we should switch default value to `homedir.Dir()`.  `useLocalConfig` is just an option.

Reply:
Thanks, I fixed.